src/ServiceControl.Audit/Auditing/AuditIngestion.cs

```diff
@@ -187,7 +187,7 @@ async Task OnMessage(MessageContext messageContext, CancellationToken cancellati
                 await channel.Writer.WriteAsync(messageContext, cancellationToken);
                 await taskCompletionSource.Task;
 
-                ingestedMessagesCounter.Add(1);
+                successfulMessagesCounter.Add(1, Telemetry.GetIngestedMessageTags(messageContext.Headers));
                 messageSize.Record(messageContext.Body.Length / 1024.0);
```

Comment:
Shouldn't this become a `messaging.message.body.size` tag?

https://opentelemetry.io/docs/specs/semconv/messaging/messaging-spans/#messaging-attributes

> [messaging.message.body.size](https://opentelemetry.io/docs/specs/semconv/attributes-registry/messaging/)	int	The size of the message body in bytes. [17]	1439	Opt-In

Reply:
I think that makes sense, will do